### ISSUE TITLE
Upgrade to ROS2

### ISSUE THREAD

**Issue by j-jacobson** [JacobsonRobotics/UnifiedRobotEnvironment#4]
Separate ticket from getting ROS1 to work. Once that works, update so that ROS2 works

Or would it be easier to start from scratch?

**Comment by j-jacobson** [comment]
Once this is done, delete the mini_ros directory.
